packages/stats/gbstats/shared/models.py

```diff
@@ -63,10 +69,14 @@ class RatioStatistic(Statistic):
 
     @property
     def mean(self):
+        if self.d_statistic.sum == 0:
+            return 0
         return self.m_statistic.sum / self.d_statistic.sum
 
     @property
     def variance(self):
+        if self.d_statistic.mean == 0 or self.n <= 1:
+            return 0
```

Comment:
Covariance below also divides by n-1, but doesn't have a check

Reply:
`covariance` is only called here by `variance` so it should never throw that error unless people are messing with the package itself (in which case I kind of prefer a divide by zero error). However, we should just be as safe as possible for the UI so I added it.